packages/devtools_app/lib/src/shared/primitives/custom_pointer_scroll_view.dart

```diff
@@ -812,9 +812,7 @@ class _RenderScrollSemantics extends RenderProxyBox {
     }
 
     _innerNode ??= SemanticsNode(showOnScreen: showOnScreen);
-    _innerNode!
-      ..isMergedIntoParent = node.isPartOfNodeMerging
-      ..rect = Offset.zero & node.rect.size;
+    _innerNode!..rect = Offset.zero & node.rect.size;
```

Comment:
Looks like the CI is complaining about using `.` instead of the cascade operator `..`.

Reply:
Thanks!